client/src/App.jsx

```diff
@@ -55,20 +59,35 @@ class App extends React.Component {
 
   componentDidMount() {
     var productId = window.location.pathname.split('/')[1];
-    //console.log('PRODUCT ID:', typeof productId);
     if (productId === null || productId === undefined || productId.length === 0){
       productId = '109';
     }
-    //console.log('HELLO HELLO')
+
+    history.listen((location, action) => {
+      console.log(`calendar is ${window.location.hash === '#availability-calendar' ? 'open' : 'closed'}`);
+      console.log(`check-in date is ${urlHelpers.getCheckInOrOutDateFromUrl(window.location.search)}`);
+      console.log(`The last navigation action was ${action}`)
+    })
+
     $.ajax({
       method: 'GET',
       url: `/${productId}/availableDates`,
       success: (dates) => {
-        //console.log('GOT SOME DATA', dates)
         this.setState({
           dates: dates
         });
 
+        $.ajax({
+          method: 'GET',
+          url: `/${productId}/minNightlyRate`,
+          success: ({minNightlyRate}) => {
+            this.setState({
+              minNightlyRate: minNightlyRate
```

Comment:
Since your variable name and property name are the same, you could use the shorthand 
`this.setState({ minNightlyRate })`